tools/system_libs.py

```diff
@@ -502,9 +503,13 @@ def create_wasm_libc_rt(libname):
   # You can provide 1 to include everything, or a comma-separated list with the ones you want
   force = os.environ.get('EMCC_FORCE_STDLIBS')
   force_all = force == '1'
-  force = set((force.split(',') if force else []) + forced)
-  if force:
-    logging.debug('forcing stdlibs: ' + str(force))
+  force_include = set((force.split(',') if force else []) + forced)
+  if force_include:
+    logging.debug('forcing stdlibs: ' + str(force_include))
+
+  # Set of libraries to include on the link line, as opposed to `force` which
+  # is the set of libraries to force include (with --whole-archive).
+  alwasy_include = set()
```

Comment:
typo: `always_include`

Reply:
My brain really can't tell the difference.., sigh.